home/jobs/OMERO-python-superbuild-build/config.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
Should this configuration be handled via `pipeline-configs.yaml` and the template config? Although I am unsure how upgrading works in that (i.e. should the generated XML files be committed)?

Reply:
I've been committing the committed files across the board recently so that I can detect if something has changed or see if something doesn't match other similar files. Happy to remove it here and delete from the index on all the servers and/or consider it a local only change.

Reply:
:+1: to removing here and keeping local changes on the devspaces.

Reply:
Pushed.